### DISCUSSION TITLE
How can I trigger this plugin if repository refs are updated for source branch?

### DISCUSSION THREAD

**Discussion by chintanvadgama** [jenkinsci/bitbucket-push-and-pull-request-plugin#232] [Q&A]
Currently, I am using this plugin along with Code-violation for automated PEP8 code check whenever pull requests are opened and this works great.

However, we see limitations with this plugin when developers do more commits on the same pull-request, or let's say the same feature branch, the Jenkins job is not getting triggered.

I Have tried all the following options but nothing seems to be working.
1. Pull request updated
2. Bitbucket Server push on the source branch
3 . Source branch of the pull request updated

**Comment by macghriogair** [comment]
Hi @chintanvadgama ,
do you mean, you want to trigger a job whenever a PullRequest is udpdated through a commit? If so, this should be already supported (cf. [BitBucketPPRPullRequestServerSourceUpdatedActionFilter](https://github.com/jenkinsci/bitbucket-push-and-pull-request-plugin/blob/master/src/main/java/io/jenkins/plugins/bitbucketpushandpullrequest/filter/pullrequest/server/BitBucketPPRPullRequestServerSourceUpdatedActionFilter.java))
Can you provide more details about your job configuration and maybe logs, please?

**Comment by chintanvadgama** [comment]
Hi @macghriogair,

Please see the attached screenshot for my job configuration. 
![screencapture-10-66-2-249-8080-job-zpytest-Infra-PEP8-Code-Check-configure-2021-01-26-01_28_42](https://user-images.githubusercontent.com/11095358/105902851-0a925600-5fd4-11eb-90b9-c56bcf61db98.png)
